agent-ini/src/IniParser.cc

```diff
@@ -770,7 +770,7 @@ int IniParser::parse_helper(IniSection&ini)
 				    if (name_to_close.empty ()) {
 					// there was no name or we did not find the specified one
 					for (it = b; it != e; ++it) {
-					    if ((*it)->getReadBy() == i)
+					    if ((*it)->getReadBy() == (int) i)
```

Comment:
well, there is risk of that size_t is unsigned, so risk of overflow At least document it why it do not harm us.